src/exposition/http.rs

```diff
@@ -54,6 +55,14 @@ mod filters {
             .and_then(handlers::human_stats)
     }
 
+    /// GET /vars.json
+    pub fn json_stats(
+    ) -> impl Filter<Extract = (impl warp::Reply,), Error = warp::Rejection> + Clone {
```

Comment:
Not important, but just for consistency with rpc-perf should we also support `metrics.json` for json_stats?

Reply:
Good idea. PR is updated to have the same paths and copies the doc comments about formats and purposes from rpc-perf. We'll want to go back and add doc comments about the other exposition endpoints too.